MoodDetectorWebApp/Views/EditDeleteTeacher/EditDeletTeacher.cshtml

```diff
@@ -0,0 +1,7 @@
+﻿
+@{
+    ViewBag.Title = "EditDeletTeacher";
+}
+
+<h2>EditDeletTeacher</h2>
```

Comment:
Savotiškas žodžio sutrumpinimas ant paskutinės raidės:D Jei rimtai tai pataisyk naming, nes meta error. `[InvalidOperationException: The view 'EditDeleteTeacher' or its master was not found or no view engine supports the searched locations.`

Reply:
Okey, dar bandau dabar padaryt, kad displayintų listą mokytojų, gal paeis taip opa iš pirmo karto